grafana/Makefile

```diff
@@ -19,26 +19,25 @@
 
 all: build
 
-VERSION?=v4.2.0
+VERSION?=v4.3.2
+DEB_VERSION?=4.3.2
 PREFIX?=gcr.io/google_containers
 ARCH?=amd64
 TEMP_DIR:=$(shell mktemp -d)
 LDFLAGS=-w -X main.version=$(VERSION) -X main.commit=unknown-dev -X main.timestamp=0 -extldflags '-static'
-DEB_BUILD=4.2.0
 KUBE_CROSS_IMAGE=gcr.io/google_containers/kube-cross:v1.8.3-1
 
-# s390x
-ALL_ARCHITECTURES=amd64 arm arm64 ppc64le
+ALL_ARCHITECTURES=amd64 arm arm64 ppc64le s390x
 ML_PLATFORMS=linux/amd64,linux/arm,linux/arm64,linux/ppc64le,linux/s390x
 
 # Set default base image dynamically for each arch
 ifeq ($(ARCH),amd64)
 	BASEIMAGE?=busybox
 	CC=gcc
 endif
-ifeq ($(ARCH),armhfrm)
+ifeq ($(ARCH),arm)
 	BASEIMAGE?=armhf/busybox
-	CC=arm-linux-gnueabi-gcc
+	CC=arm-linux-gnueabihf-gcc
```

Comment:
I am not familiar with `armhf`, so i have no idea about this change. Others seem good to me.